src/features/inventory/button.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+
+import Button from '../../components/button';
+
+class InventoryButton extends Component {
```

Comment:
It might make sense to implement this using hooks. Other than using `componentDidMount` and `componentWillUnmount` there isn't much reason for this to be a class, and both of those can be replaced with `useEffect()`.